docs/docs/install/truenas.md

```diff
@@ -30,6 +30,8 @@ You can organize these as one parent with seven child datasets, for example `mnt
 
 :::info Permissions
 The **pgData** dataset must be owned by the user `netdata` (UID 999) for postgres to start. The other datasets must be owned by the user `root` (UID 0) or a group that includes the user `root` (UID 0) for immich to have the necessary permissions.
+
+The **library** dataset must have [ACL mode](https://www.truenas.com/docs/core/coretutorials/storage/pools/permissions/#access-control-lists) set to `Passthrough` if you plan using a [storage template](/docs/administration/storage-template.mdx) and the dataset is configured for network sharing (its ACL type is set to `SMB/NFSv4`). When the template is applied and files need to be moved from **uploads** to the **library**, immich performs `chmod` internally and needs to be allowed to execute the command.
```

Comment:
```suggestion
The **library** dataset must have [ACL mode](https://www.truenas.com/docs/core/coretutorials/storage/pools/permissions/#access-control-lists) set to `Passthrough` if you plan on using a [storage template](/docs/administration/storage-template.mdx) and the dataset is configured for network sharing (its ACL type is set to `SMB/NFSv4`). When the template is applied and files need to be moved from **upload** to **library**, immich performs `chmod` internally and needs to be allowed to execute the command.
```

Reply:
I've applied both changes, of course, but I wonder a bit about the singular in "uploads" - it references the folder/dataset that immich uses. It always seems to be plural on this page and in this context. If you weren't happy with singular above after all, just say a word.

Reply:
I was just going based off the code and release notes.

https://github.com/immich-app/immich/blob/fa9bb8074cec18cbaa2d1df29e48e8f4cbec5e9d/server/src/enum.ts#L156

https://github.com/immich-app/immich/releases/tag/v1.92.0
* `upload` instead of `uploads`

Reply:
Or we can go the other way around and change the other locations to a singular. I don't mind as it would be just 2 more places:

![image](https://github.com/user-attachments/assets/05b60a00-2a7a-4882-9046-6ba789c4a8a5)

![image](https://github.com/user-attachments/assets/eb3fef16-b73a-4d1d-952b-1a0bb05e6992)